locales/de/zodiac.json

```diff
@@ -0,0 +1,62 @@
+{
+	"aries": {
+		"name": "Widder",
+		"element": 0,
+		"stone": "Blutstein"
+	},
+	"taurus": {
+		"name": "Stier",
+		"element": 1,
+		"stone": "Saphir "
+	},
+	"gemini": {
+		"name": "Zwilling",
+		"element": 2,
+		"stone": "Achat"
+	},
+	"cancer": {
+		"name": "Krebs",
+		"element": 3,
+		"stone": "Smaragd"
+	},
+	"leo": {
+		"name": "Löwe",
+		"element": 0,
+		"stone": "Onyx"
+	},
+	"virgo": {
+		"name": "Jungfrau",
+		"element": 1,
+		"stone": "Karneol"
+	},
+	"libra": {
+		"name": "Waage",
+		"element": 2,
+		"stone": "Chrysolith"
+	},
+	"scorpio": {
+		"name": "Skorpion",
+		"element": 3,
+		"stone": "Beryll"
+	},
+	"sagittarius": {
+		"name": "Schütze",
+		"element": 0,
+		"stone": "Topas"
+	},
+	"capricorn": {
+		"name": "Steinbock",
+		"element": 1,
+		"stone": "Rubin"
+	},
+	"aquarius": {
+		"name": "Wassermann",
+		"element": 2,
+		"stone": "Granat"
```

Comment:
Is it the right translation?

- https://de.wikipedia.org/wiki/Granatgruppe
- https://en.wikipedia.org/wiki/Garnet

Reply:
"Granat" is correct.

Literally "Granatgruppe" means "Group of Garnet" which indicates all the compound of minerals, the stone in German is commonly called "Granat".
I don't know why here: https://de.wikipedia.org/wiki/Granatgruppe abbreviates it to "Granate" which literally means "Grenade", but I'm pretty sure the stone is called "Granat"